### PR TITLE
Handle plugin compatibility issue with other flutter notification plugins

### PR DESCRIPTION
**Pull request by theideasaler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ MC_MID="<YOUR_SFMC_MID>"
 MC_SERVER_URL="<YOUR_SFMC_URL>"
 ```
 ## Setup iOS 
-
+### Update `info.plist`
```
Comment (sefidgaran) [inline README.md]: Move it to line 64, and replace it with '### Provision for Push'.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,22 @@ Note: Please add UIBackgroundModes keys into your info.plist file as below:
 	</array>
 ```
 
+### Setup `UNUserNotificationCenter` delegate
```
Comment (sefidgaran) [inline README.md:74]: Please update src/example project as well. thanks

File: README.md (hunk 1 above)
Comment (sefidgaran) [reply README.md]: Please update src/README.md file as well. thanks
README.md and src/README.md should be equal.

File: src/ios/Classes/SwiftSfmcPlugin.swift [hunk 3]
```diff
@@ -193,15 +188,23 @@ public class SwiftSfmcPlugin: NSObject, FlutterPlugin, MarketingCloudSDKURLHandl
...
+            MarketingCloudSDK.sharedInstance().sfmc_setNotificationRequest(response.notification.request)
+            completionHandler()
+        }
     }
 
     // The method will be called on the delegate only if the application is in the foreground. If the method is not implemented or the handler is not called in a timely manner then the notification will not be presented. The application can choose to have the notification presented as a sound, badge, alert and/or in the notification list. This decision should be based on whether the information in the notification is otherwise visible to the user.
     @available(iOS 10.0, *)
     public func userNotificationCenter(_ center: UNUserNotificationCenter, willPresent notification: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
-        completionHandler(.alert)
+        let userInfo = notification.request.content.userInfo
```
Comment (sefidgaran) [inline src/ios/Classes/SwiftSfmcPlugin.swift:204]: Please make sure SFMC push notifications and in app messages along with deep linking work as previous version. thanks

**Review by sefidgaran — CHANGES_REQUESTED** [review]
@theideasaler , left a few comments. thanks!

File: README.md (hunk 1 above)
Comment (theideasaler) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (theideasaler) [reply README.md:74]: fixed

File: src/ios/Classes/SwiftSfmcPlugin.swift (hunk 3 above)
Comment (theideasaler) [reply src/ios/Classes/SwiftSfmcPlugin.swift:204]: checked, work as previous

File: README.md (hunk 1 above)
Comment (sefidgaran) [reply README.md]: My first comment is not resolved, I will change it by myself.

**Review by sefidgaran — APPROVED** [review]

**Pull request merged**